CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- `PanelSidebar` is new sidebar with custom side panels and a new `PanelSideBarLayoutContext`, also for older sidebar
+- `collapseIconOnly` to `PanelItem` in order to collapse `PanelSideBarItem` only by clicking on chevron icon and label can work as link
+
+### Removed
+
+- `DateHandler` utils as it belongs to another package and will be replaced in the future
```

Comment:
Maybe we should mention in the changelog that the drop of the DateHandler is a breaking change?
Are there other breaking changes to mention maybe?